custom_components/tuya_local/devices/bonaire_heater.yaml

```diff
@@ -0,0 +1,119 @@
+name: Bonaire Electric Heater
+products:
+  - id: xto0qf9xem5bi7sj
+    name: Bonaire Electric Heater HPC5200020
+primary_entity:
+  entity: climate
+  translation_only_key: heater
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+        - dps_val: false
+          value: "off"
+    - id: 2
+      name: temperature
+      type: integer
+      unit: C
+      range:
+        min: 5
+        max: 35
+    - id: 3
+      name: current_temperature
+      type: integer
+    - id: 4
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: low
+          value: eco
+        - dps_val: high
+          value: boost
+        - dps_val: af
+          value: away
+    - id: 12
```

Comment:
You are probably copying one of the older configs here.
It is generally better to separate any error flags out to a separate binary_sensor with class: problem.  A good example is [taurus_agadir_heater](https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/devices/taurus_agadir_heater.yaml#L131)

Reply:
That makes sense!